### ISSUE TITLE
Level of Detail

### ISSUE THREAD

**Issue by minghia** [cedricpinson/osgexport#27]
Does the latest version of osgexport handle the Level of Detail feature that is now present in Blender 2.70a?

**Comment by klemen666** [comment]
Hi.
Do you think that exporter will support LOD from Blender? We are using Blender 2.79 and would really need that feature.
Regards,
Klemen

**Comment by cedricpinson** [comment]
I dont think so, if you need it should you should fork and the necessary code / or make a pull request

**Comment by matevz** [comment]
Hi guys.
I added support for LODs here:
https://github.com/matevz/osgexport-lod
Please test and provide feedback.

**Comment by mp3butcher** [comment]
next passenger: please make a pr
